cht-dolly-v2/docker/gpu/Dockerfile

```diff
@@ -1,22 +1,15 @@
 FROM huggingface/transformers-pytorch-gpu:4.28.1
-
-ARG MODEL_ID
-
-RUN pip install "accelerate>=0.16.0,<1"
-
 WORKDIR /usr/src/app/
 
-COPY requirements.txt ./
-
-RUN pip install --no-cache-dir -r ./requirements.txt --upgrade pip
+COPY requirements.txt .
+RUN pip install "accelerate>=0.16.0,<1"
+RUN pip install --no-cache-dir -r requirements.txt
 
+ARG MODEL_ID
 COPY download.py .
-
-RUN python3 download.py --model $MODEL_ID
+RUN python download.py --model $MODEL_ID
 
 COPY . .
-
 ENV MODEL_ID=$MODEL_ID
-ENV DEVICE=auto
```

Comment:
is this required? different Dockerfiles use one of:

- `cpu`
- `cuda`
- `auto`
- nothing

Reply:
So when explicitly `cpu` or `cuda` is mentioned only these two cases should be required to make the model load on specified device on first try, I think `auto` cases can be removed.

Reply:
Yep